tests/sqlparser_mysql.rs

```diff
@@ -100,6 +100,35 @@ fn parse_show_columns() {
     }
 }
 
+#[test]
+fn parse_show_create() {
+    let obj_name = ObjectName(vec![Ident::new("myident")]);
+
+    assert_eq!(
```

Comment:
I wonder if it would be possible to check all the possible variants here (perhaps with a loop / expected values)?